### PR TITLE
Migration tool

### PR DESCRIPTION
**Pull request by MBishop17**
Integrating instructions for migration tool into migration guide.

### CODE REVIEW

**Comment by rawmind0** [comment]
Reference https://github.com/rancher/rancher/issues/15454

File: content/rancher/v2.x/en/v1.6-migration/_index.md [hunk 1]
```diff
@@ -63,7 +62,77 @@ As a Rancher 1.6 user who's interested in moving to 2.0, how should you get star
...
 
-## 2. Migrate Applications
+## 2. Run Migration Tools
+
+To help with migration from 1.6 to 2.0, Rancher has developed a migration tool. Running this tool will help you check if your Rancher 1.6 applications can be migrated to 2.0. If an application can't be migrated, the tool will help you identify what's lacking. 
+
+This tool will:
+
+- Accept Docker Compose config files (i.e., `docker-compose.yml` and `rancher-compose.yml`) that you've exported from your Rancher 1.6 Stacks.
+- Output a list of constructs present in the Compose files that cannot be supported by Kubernetes in Rancher 2.0. These constructs require special handling or parameters that cannot be converted to Kubernetes YAML, even using tools like Kompose.
```
Comment (prachidamle) [inline content/rancher/v2.x/en/v1.6-migration/_index.md]: The last sentence seems missing 'are'? "These constructs require special handling or _are_ parameters ...'

File: content/rancher/v2.x/en/v1.6-migration/_index.md [hunk 2]
```diff
@@ -63,7 +62,77 @@ As a Rancher 1.6 user who's interested in moving to 2.0, how should you get star
...
+To use the Migration Tool, run the command below while pointing to the compose files exported from each stack that you want to migrate. If you want to migrate multiple stacks, you'll have to re-run the command for each pair of compose files that you exported.
+
+#### Usage
+
+You can run the Migration Tool by entering the following command, replacing each placeholder with the absolute path to your Stack's compose files.
+
+```
+migration-tools --docker-file <DOCKER_COMPOSE_ABSOLUTE_PATH> --rancher-file <RANCHER_COMPOSE_ABSOLUTE_PATH>
+```
+
```
Comment (prachidamle) [inline content/rancher/v2.x/en/v1.6-migration/_index.md]: If you do not provide the `--docker-file <DOCKER_COMPOSE_ABSOLUTE_PATH> --rancher-file <RANCHER_COMPOSE_ABSOLUTE_PATH>` options, the tool will look for these files in the folder where the tool is running from.

**Comment by prachidamle** [comment]
LGTM - 
@deniseschannon @MBishop17 
We should have a follow-up PR to add docs about the new updates to the migration-tool made by @rawmind0

**Pull request merged**